src/GovUk.Education.ExploreEducationStatistics.Admin/Requests/ReleaseStatusCreateRequest.cs

```diff
@@ -47,4 +47,6 @@ public record ReleaseStatusCreateRequest
     }
 
     [PartialDateValidator] public PartialDate? NextReleaseDate { get; init; }
+
+    public bool? UpdatePublishedDate { get; init; }
```

Comment:
I made it optional because the `UpdatePublishedDate` field is conditionally rendered depending on approval status and whether it's an amendment or not.

When moving between Draft -> Higher Review or from any status back to Draft, or when the release is not an amendment, it's not expected in the request.